README.md

```diff
@@ -28,10 +28,19 @@ Before using the boilerplate, we recommend you to go through the documentation o
 3. [Web Performance](https://www.aem.live/developer/keeping-it-100)
 4. [Markup, Sections, Blocks, and Auto Blocking](https://www.aem.live/developer/markup-sections-blocks)
 
-## Config Service Setup
+## Initial Setup
+
+The boilerplate assumes you already have an AEM org and will onboard a new site via config service. If you do not have an org, you will need to contact Adobe to have one created, or you can do the following:
```

Comment:
@tripodsan not sure if we can just say "ask Adobe" or if there is a more specific process for this.

Reply:
no. there isn't.

Reply:
And can I point them somewhere in a link to "contact adobe" like... Discord, slack?

Reply:
Pointed to discord for now.